src/pages/Main/MainFilter.js

```diff
@@ -0,0 +1,340 @@
+import React, { useRef, useEffect, useState } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+const MainFilter = ({ setIsOpenModal }) => {
+  const [selected, setSelected] = useState({
+    bathroom: '상관없음',
+    bed: '상관없음',
+    bath: '상관없음',
+  });
+  const [filterForm, setFilterForm] = useState({
+    minprice: '',
+    maxprice: '',
+    all: '',
+    priv: '',
+    share: '',
+  });
+
+  const navigate = useNavigate();
+
+  // const queryString = `/products/?search${
+  //   filterForm.minprice ? `min_price=${filterForm.minprice} &` : ''
+  // }${filterForm.maxprice ? `max_price=${filterForm.maxprice} &` : ''}${
+  //   filterForm.all ? `all=${filterForm.all} &` : ''
+  // }${filterForm.priv ? `priv=${filterForm.priv} &` : ''}${
+  //   filterForm.share ? `share=${filterForm.share} &` : ''
+  // }${selected.bathroom && `bathroom=${selected.bathroom} &`}${
+  //   selected.bath && `bath=${selected.bath} &`
+  // }${selected.bed && `bed=${selected.bed}`}`;
+
+  const queryString = {
```

Comment:
실제 데이터는 string이 아니나, queryString이라는 변수명을 사용하고 있습니다!
object를 나타내는 데이터는 보통 접미사로 Map을 사용합니다

또 자주 사용하는 접미사로는 array에 XXXList 가 있을 것 같습니다.

```suggestion
  const queryMap = {
```